scripts/experiment_1/functions/vae.py

```diff
@@ -0,0 +1,322 @@
+# By Alexandra Lee
+# (updated October 2018)
+#
+# Encode gene expression data into low dimensional latent space using
+# Tybalt with 2-hidden layers
+
+import os
+import argparse
+import pandas as pd
+import tensorflow as tf
+
+# To ensure reproducibility using Keras during development
+# https://keras.io/getting-started/faq/#how-can-i-obtain-reproducible-results-using-keras-during-development
+import numpy as np
+import random as rn
+
+from keras.layers import Input, Dense, Lambda, Layer, Activation
+from keras.layers.normalization import BatchNormalization
+from keras.models import Model, Sequential
+from keras import metrics, optimizers
+from keras.callbacks import Callback
+
+from functions.helper_ae import sampling_maker, CustomVariationalLayer, WarmUpCallback
+
+
+def tybalt_2layer_model(
+        learning_rate,
+        batch_size,
+        epochs,
+        kappa,
+        intermediate_dim,
+        latent_dim,
+        epsilon_std,
+        rnaseq,
+        base_dir,
+        analysis_name):
+    """
+    Train 2-layer Tybalt model using input dataset
+
+    Arguments:
+    -- learning_rate: step size used for gradient descent. In other words, its how quickly the  methods is learning
```

Comment:
its -> it's

Reply:
fixed.  At some point I will want to update the text in these descriptions to be cleaner